webapp/shop/cred/views.py

```diff
@@ -532,6 +532,15 @@ def cred_shop(**kwargs):
     return flask.render_template("credentials/shop/index.html")
 
 
+@shop_decorator(area="cube", permission="user", response="html")
+@canonical_staff()
+def cred_shop_keys(**kwargs):
+    cue_price = get_cue_products(type="keys").json
+    return flask.render_template(
+        "credentials/shop/keys.html", cue_price=cue_price[0]
```

Comment:
This should probably be named `cue_key_product` or something similar rather than `cue_price`?